sklearn/metrics/cluster/supervised.py

```diff
@@ -870,6 +870,10 @@ def normalized_mutual_info_score(labels_true, labels_pred,
     normalizer = _generalized_average(h_true, h_pred, average_method)
     # Avoid 0.0 / 0.0 when either entropy is zero.
     normalizer = max(normalizer, np.finfo('float64').eps)
+
+    # Handling sparse input
+    if normalizer < np.finfo('float64').eps:
+        return 0
```

Comment:
The coverage report seems to indicate that no test can trigger this case.

Reply:
Ah apologies @ogrisel I realise this is not such a good approach as anyways :

```normalizer = max(normalizer, np.finfo('float64').eps)```

the minimum for `normalizer` cannot be less than epsilon.

Reply:
So I don't think this control structure would ever be triggered